guides/common/modules/con_performance-tuning-quickstart.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Performance_Tuning_Quickstart_{context}"]
-= Performance tuning quickstart
+= Performance tuning quick start
```

Comment:
Please also adjust the anchor and file name.

Reply:
Will do.